chess_gui.py

```diff
@@ -118,12 +128,38 @@ def main():
     player_clicks = []  # keeps track of player clicks (two tuples)
     valid_moves = []
     game_over = False
+    move_num = 0
+    checks_amount = 0
+    knight_moves = 0
+    white_moves = 0
+    black_moves = 0
+    white_moves_bool = False
+    black_moves_bool = False
+    now_happend = [Logs.NOTHING]
 
     ai = ai_engine.chess_ai()
     game_state = chess_engine.game_state()
-    if human_player is 'b':
+    if human_player == 'b':
+        logging.info("The computer started playing")
         ai_move = ai.minimax_black(game_state, 3, -100000, 100000, True, Player.PLAYER_1)
-        game_state.move_piece(ai_move[0], ai_move[1], True)
+        move_num += 1
+        for i in range(0, len(game_state.board)):
+            for j in range(0, len(game_state.board[i])):
+                if game_state.board[i][j] != -9:
```

Comment:
Try to avoid using 'i' and 'j' if better names are avalible

Reply:
Got fixed